cmd/ui/src/views/Explore/EdgeInfo/EdgeInfoContent.test.tsx

```diff
@@ -87,4 +124,14 @@ describe('EdgeInfoContent', () => {
             screen.queryByText('An unexpected error has occurred. Please refresh the page and try again.')
         ).not.toBeInTheDocument();
     });
+    test('Selecting an edge with a Computer target node that haslaps is enabled shows correct Windows Abuse text', async () => {
```

Comment:
Thanks for adding this test! Would you please be so kind as to add an additional test where the target computer node does not have a `haslaps` property and assert that the `windowsabuse-computer-has-laps-text` element is not rendered?

Reply:
Thanks for the suggestion Uli, just added a new test for that. What i did was test that the alternate text was rendered ( was easier to implement and felt that it still tested what we wanted), let me know if that works or if you prefer that the haslaps one is not rendered.

Reply:
Looks good! Thank you